rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/core/properties/PAResourceManagerProperties.java

```diff
@@ -247,7 +247,32 @@
      * The approach is best effort and Node hostname is not considered.
      * As a result, Nodes are not necessarily locked on the same host.
      */
-    RM_NODES_LOCK_RESTORATION("pa.rm.nodes.lock.restoration", PropertyType.BOOLEAN, "true");
+    RM_NODES_LOCK_RESTORATION("pa.rm.nodes.lock.restoration", PropertyType.BOOLEAN, "false"),
```

Comment:
probably this property change was a mistake

Reply:
Yes it is a mistake. Addressed.